README.md

```diff
@@ -53,7 +55,7 @@ $ npm install --global resin-preload
 ## Requirements
 
 - [Node](https://nodejs.org)
-- [Docker](https://www.docker.com) tested on 1.12.6 and up but 17.04 or up is recommended especially on macOS
+- [Docker](https://www.docker.com) tested on 1.12.6 and up but 17.04 or up is recommended especially on macOS, [docker-toolbox](https://www.docker.com/products/docker-toolbox) is not supported.
```

Comment:
Should this also link to the `Version Compatibility` section in this README, as there's a previous version which still supports that?

Reply:
Added